examples/shared_samples/scope.rb

```diff
@@ -29,6 +37,15 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::C7000::Scope
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::Scope
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::C7000::Scope
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::Scope
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::C7000::Scope
```

Comment:
It should be OneviewSDK::API800::Synergy::Scope

Reply:
Updated as mentioned